spring-boot-cxf-jaxws/pom.xml

```diff
@@ -48,6 +48,15 @@
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-rt-features-logging</artifactId>
         </dependency>
+        <!-- Camel -->
```

Comment:
Was the test for plain cxf? do we really want to introduce Camel? @lnearlol @mcarlett @luigidemasi

Reply:
@lnearlol sorry I didn't know that this example should not have the Camel dependencies, so this way is not doable.